src/v3/src/types/schema.ts

```diff
@@ -248,10 +248,9 @@ export interface PasswordRequirementsElement extends UISchemaElement {
 
 export interface LinkElement extends UISchemaElement {
   type: 'Link';
-  options: ActionOptions & {
+  options: Omit<ButtonElement['options'], 'type'> & {
```

Comment:
Perhaps we should just define our own type at this point rather than copy from `ButtonElement`?